app/presenters/iiif_print/iiif_manifest_presenter_behavior.rb

```diff
@@ -10,5 +10,20 @@ def manifest_metadata
     def search_service
       Rails.application.routes.url_helpers.solr_document_iiif_search_url(id, host: hostname)
     end
+
+    # OVERRIDE Hyrax 3x, avoid nil returning to IIIF Manifest gem
+    # @see https://github.com/samvera/iiif_manifest/blob/c408f90eba11bef908796c7236ba6bcf8d687acc/lib/iiif_manifest/v3/manifest_builder/record_property_builder.rb#L28
+    ##
+    # @return [Array<Hash{String => String}>]
+    def sequence_rendering
+      Array(try(:rendering_ids)).map do |file_set_id|
+        rendering = file_set_presenters.find { |p| p.id == file_set_id }
+        return [] unless rendering
```

Comment:
specifically added this line, to return [] instead of nil. Otherwise iiif_manifest translates it to [nil] and breaks code.